lib/package.rb

```diff
@@ -11,7 +11,7 @@ class Package
 
   boolean_property = %i[conflicts_ok git_clone_deep git_fetchtags gnome is_fake is_musl is_static
                         no_compile_needed no_compress no_env_options no_fhs no_git_submodules
-                        no_links no_patchelf no_shrink no_strip no_zstd patchelf]
+                        no_links no_patchelf no_shrink no_strip no_lto no_zstd patchelf]
```

Comment:
Everything is listed in alphabetical order to prevent duplicates.

Reply:
Ah, let me fix that!